benchmarks/graphics_pipeline/main.cpp

```diff
@@ -783,6 +835,62 @@ void ProjApp::Setup()
     }
 }
 
+void ProjApp::KeyDown(ppx::KeyCode key)
+{
+    mPressedKeys.insert(key);
+}
+
+void ProjApp::KeyUp(ppx::KeyCode key)
+{
+    mPressedKeys.erase(key);
+}
+
+void FreeCamera::Move(MovementDirection dir, float distance)
```

Comment:
maybe move this above all the ProjApp function definitions? So the function implem order matches the definition order.

Reply:
Sound good to me, added before ProjApp method implementations.